awx/api/views/__init__.py

```diff
@@ -536,10 +540,12 @@ class InstanceGroupAccessList(ResourceAccessList):
 
 
 class InstanceGroupObjectRolesList(SubListAPIView):
+    deprecated = True
```

Comment:
Are we really going to deprecate everything?  That seems pretty hostile to community users.

(also, this one seems to have `deprecated = True` twice.)

Reply:
Talking about deprecation, if I understood you correctly `access_list` should be marked as deprecated as well right?

Reply:
Put up https://github.com/ansible/awx/pull/15068, which should address the things mentioned in both of the above 2 comments.

And yes, these will be deprecated. Obviously we won't just go break the UI. We will push the new UI to the community somehow and stop maintaining the old UI. Some time after that, yes, we will remove these endpoints, and we will delete the old `Role` tables that are used for the backward compatibility. Everything you could do with these you can do with the new endpoints.